src/modules/ekf2/EKF/optical_flow_control.cpp

```diff
@@ -59,7 +59,7 @@ void Ekf::controlOpticalFlowFusion(const imuSample &imu_delayed)
 
 	// Accumulate autopilot gyro data across the same time interval as the flow sensor
 	const Vector3f delta_angle(imu_delayed.delta_ang - (getGyroBias() * imu_delayed.delta_ang_dt));
-	if (_delta_time_of < 0.1f) {
+	if (_delta_time_of < 0.5f) {
```

Comment:
This just seems really high, maybe we start with doubling it (consistent with the max below)?